tests/test_data/test_pipelines/test_formatting.py

```diff
@@ -0,0 +1,91 @@
+import copy
+import os.path as osp
+
+import numpy as np
+import pytest
+import torch
+
+import mmcv
```

Comment:
you can replace `import mmcv` with `from mmcv.image.io import imread` to resolve CI